examples/listbox/js/listbox.js

```diff
@@ -497,14 +507,14 @@ aria.Listbox.prototype.updateScroll = function () {
 };
 
 /**
- * @desc
+ * @description
  *  Enable/disable the up/down arrows based on the activeDescendant.
  */
 aria.Listbox.prototype.checkUpDownButtons = function () {
   var activeElement = document.getElementById(this.activeDescendant);
 
   if (!this.moveUpDownEnabled) {
-    return false;
+    return;
```

Comment:
These were functional changes flagged by the `return` rule. Since the value was never read, I matched the other void returns